### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+[Full Changelog](https://github.com/theforeman/puppet-pulpcore/compare/3.4.0...4.0.0)
+
+**Breaking changes:**
+
+- Support Pulp 3.13, drop earlier versions [\#202](https://github.com/theforeman/puppet-pulpcore/pull/202) ([ekohl](https://github.com/ekohl))
+
+**Implemented enhancements:**
+
+- Support 3.14, drop 3.13 [\#208](https://github.com/theforeman/puppet-pulpcore/pull/208) ([ehelms](https://github.com/ehelms))
```
Comment (ekohl) [inline CHANGELOG.md:13]: Can we also make this a breaking change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +35,10 @@
 
 - Fixes [\#32622](https://projects.theforeman.org/issues/32622): Include StdEnvVars, ExportCertData SSL options in Apache [\#193](https://github.com/theforeman/puppet-pulpcore/pull/193) ([ehelms](https://github.com/ehelms))
 
+## [2.2.3](https://github.com/theforeman/puppet-pulpcore/tree/2.2.3) (2021-05-19)
+
+[Full Changelog](https://github.com/theforeman/puppet-pulpcore/compare/3.3.0...2.2.3)
```
Comment (ekohl) [inline CHANGELOG.md:40]: I'd leave this out. Probably also means you need to do a change to HISTORY.md to keep it form showing up in the future.

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by ehelms** [comment]
Updated with the log level change

**Review by wbclark — APPROVED** [review]
ACK

**Pull request merged**